config/src/doc.rs

```diff
@@ -12,6 +12,8 @@ pub struct DocConfig {
     pub title: String,
     /// Path to user provided `book.toml`.
     pub book: PathBuf,
+    /// Path to user provided welcome markdown (default is Readme.md)
+    pub homepage: PathBuf,
```

Comment:
ah so this is expected to be set in the toml file?

I think this should be an Option